toqito/channels/tests/test_bitflip.py

```diff
@@ -0,0 +1,81 @@
+"""Test cases for the bit-flip channel."""
+
+import numpy as np
+import pytest
+
+from toqito.channels import bitflip
+
+
+def test_kraus_operators():
+    """Test if the function returns correct Kraus operators for given probability."""
+    prob = 0.3
+    kraus_ops = bitflip(prob=prob)
+    expected_k0 = np.sqrt(1 - prob) * np.eye(2)
+    expected_k1 = np.sqrt(prob) * np.array([[0, 1], [1, 0]])

```

Comment:
You can create a list of these two and directly compare the calculated output to the expected output.